Vaccine product and manufacturer terminology improvements (#132)
dvci/vaccine-credential-ig: input/includes/immunization.md

For example, if there are multiple vaccines for a specific disease with different numbers of doses, different schedules for doses, or meaningful differences in effectiveness, Issuers SHOULD provide manufacturer using one of the approaches described above.

Note manufacturer and lot number may be recorded together in vaccine records. To conform with this IG, manufacturer SHALL NOT appear in the `lotNumber` element; instead, Issuers SHALL apply the above conformance criteria to determine when and how manufacturer can be identified.

#### CVX and MVX codes

When vaccine and manufacturer are provided using US-centric terminology (CVX and MVX, respectively) for COVID-19 vaccinations, CDC [provides a list](https://www.cdc.gov/vaccines/programs/iis/COVID-19-related-codes.html) that includes "Sale Proprietary Name" (e.g., `Moderna COVID-19 Vaccine`). The "Sale Proprietary Name" or other trade name SHALL NOT be included in FHIR resources, but MAY be used by actors when producing human-readable representations of these resources.

> Note that as of May 2021, CVX implicitly identifies specific vaccine products and manufacturers **only for COVID-19**. For other diseases, CVX typically identifies just the target disease and vaccine type. Issuers that only populate `vaccineCode` with a CVX code therefore MAY provide a manufacturer separately for COVID-19 vaccines, and SHOULD provide a manufacturer

Reviewer: ```suggestion
Note that as of May 2021, CVX implicitly identifies specific vaccine products and manufacturers **only for COVID-19**. For other diseases, CVX typically identifies just the target disease and vaccine type. Issuers that only populate `vaccineCode` with a CVX code therefore SHOULD NOT populate a `manufacturer` for COVID-19 vaccinations, but MAY provide a manufacturer for non-COVID-19 vaccinations.
```

Looks like this sentence was truncated, and I'm trying to infer what it meant. I hope/expect we can strongly discourage ("SHOULD NOT") issuers from including an explicit manufacturer when there's already an implicit manufacturter known.

Author: Thanks Josh -- this was the intent here.
